<launch>

    <!-- <arg name="enable_debug" /> -->
    <node pkg="nearness_control" type="marble_joy_interface_node" name="joy_interface_node" output="screen">
        <!-- TOPICS -->
        <remap from="marble_joy_interface/joy" to="joy"/>
        <remap from="marble_joy_interface/estop_cmd" to="estop_cmd"/>
        <remap from="marble_joy_interface/radio_reset_cmd" to="radio_reset_cmd"/>
    </node>

</launch>
